all.js

```diff
@@ -8,6 +8,17 @@ var simssh = require('simple-ssh');
 var scp2 = require('scp2')
 var biHelper = require('./biHelper.js');
 
+// [REVIEW] we should have only one uploadFiles, either uploadFiles, or uploadFilesViaScp
```

Comment:
Let's remove 'uploadFiles' and use 'uploadFilesViaScp' only, since the later support folder creation.